exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -107,6 +107,9 @@
   <p>FOREMAN: ${model.getProfile().getForeman().getAddress()}</p>
   <p>TOTAL FRAGMENTS: ${model.getProfile().getTotalFragments()}</p>
   <p>DURATION: ${model.getProfileDuration()}</p>
+  <p>&nbsp;&nbsp;&nbsp;&nbsp;PLANNING: ${model.getPlanningDuration()}</p>
+  <p>&nbsp;&nbsp;&nbsp;&nbsp;QUEUED: ${model.getQueuedDuration()}</p>
+  <p>&nbsp;&nbsp;&nbsp;&nbsp;EXECUTION: ${model.getExecutionDuration()}</p>
```

Comment:
Thanks for the indent. But, using nbsp is old-school. Please create a style that sets the left margin to, say 4em.

Also, as long as we are here, might as well clean up the labels. Shift from UPPER CASE to Title Case.

And, for extra credit, change the whole mess to a table so that the values are all aligned in a nice column.